packages/interpreter/src/interpreter.js

```diff
@@ -2,7 +2,7 @@ export function main() {
   let root = window.document.getElementById("main");
   if (root != null) {
     window.interpreter = new Interpreter(root);
-    window.rpc.call("initialize");
+    window.ipc.postMessage(JSON.stringify({ type: "initialize" }));
```

Comment:
this is a api change ?

Reply:
https://github.com/tauri-apps/wry/blob/dev/CHANGELOG.md

Yes this PR upgrades to wry 0.13.1

Some new features with wry too